lib/admin/class-groups-admin-users.php

```diff
@@ -154,7 +154,7 @@ public static function restrict_manage_users() {
 		if ( ( $pagenow == 'users.php' ) && empty( $_GET['page'] ) ) {
 			// groups select
 			$groups_table = _groups_get_tablename( 'group' );
-			if ( $groups = $wpdb->get_results( "SELECT * FROM $groups_table ORDER BY name" ) ) {
+			if ( $groups = apply_filters( 'groups_manage_users', $wpdb->get_results( "SELECT * FROM $groups_table ORDER BY name" ) ) ) {
```

Comment:
We need to review this filter `groups_manage_users` because it does not adhere to our naming conventions, noted for review and inclusion after adjustment.